transcendence-app/src/user/infrastructure/db/user.entity.ts

```diff
@@ -10,16 +10,38 @@ export enum userKeys {
   AVATAR_Y = '"avatarY"',
 }
 
+type UserEntityData = {
```

Comment:
what do you need this type for?

Reply:
We use this type to instantiate the CreateMessageWithUserEntity https://github.com/42AGV/ft_transcendence/blob/c463830a0d1a7e3914c087cb826b77234afbf7eb/transcendence-app/src/chat/infrastructure/db/chat-room-message-with-user.entity.ts#L22